server/lib/queries.js

```diff
@@ -48,12 +48,16 @@ module.exports = function(sequelize) {
   /**
    * Get top collectives based on total donations
    */
-  const getGroupsByTag = (tag, limit, excludeList, minTotalDonation, randomOrder) => {
-    var excludeClause = ''
+  const getGroupsByTag = (tag, limit, excludeList, minTotalDonation, randomOrder, orderBy, orderDir, offset) => {
+    var tagClause = '';
+    var excludeClause = '';
     var minTotalDonationClause = '';
     var orderClause = 'BY t."totalDonations"';
-    if (randomOrder) {
-      orderClause = 'BY random()'
+    var orderDirection = (orderDir === 'asc') ? 'ASC' : 'DESC';
```

Comment:
This was done with backward-compatibility in mind. 

It assumes that `orderDir` MAY be undefined, so instead we default to `desc`, which was the original sort direction. We only change direction to `asc` if it is explicitly passed.

This was done since the method is used elsewhere, and I wanted to preserve the method's usage as designed originally because I feel this mitigates the introduction of bugs.

I tippy-toed around existing/expected functionality